include/rabbit.hrl

```diff
@@ -38,6 +38,8 @@
 -record(permission, {configure, write, read}).
 -record(user_vhost, {username, virtual_host}).
 -record(user_permission, {user_vhost, permission}).
+-record(topic_permission_key, {user_vhost, name}).
```

Comment:
I think this should be `exchange` instead. "Topic name" as a concept is a bit ambiguous in the current codebase (and AMQP 0-9-1). So until we get to the promised "protocol agnostic core" land it'd probably be clearer as `exchange`, in particular since the docs use the same name.

Note that CLI already uses "exchange" everywhere.